runtime/autoload/man.vim

```diff
@@ -242,18 +243,26 @@ function! man#complete(arg_lead, cmd_line, cursor_pos) abort
     let name = a:arg_lead
     let sect = tolower(args[1])
   endif
-  " We remove duplicates incase the same manpage in different languages was found.
-  return uniq(sort(map(globpath(s:mandirs,'man?/'.name.'*.'.sect.'*', 0, 1), 's:format_candidate(v:val, sect)'), 'i'))
+  return s:complete(sect, sect, name)
 endfunction
 
-function! s:format_candidate(path, sect) abort
+function! s:complete(sect, psect, name) abort
+  let old_fic = &fileignorecase
+  let &fileignorecase = &wildignorecase
```

Comment:
If the user can set 'fileignorecase' already, why should 'wildignorecase' drive this behavior?